db/db_test.go

```diff
@@ -0,0 +1,87 @@
+package db_test
+
+import (
+	"log"
+	"os"
+	"testing"
+
+	"github.com/mikloslorinczi/infra-exec/common"
+
+	"github.com/mikloslorinczi/infra-exec/db"
+	"github.com/stretchr/testify/require"
+	"github.com/stretchr/testify/suite"
+)
+
+type DBTestSuite struct {
+	suite.Suite
+	*require.Assertions
+	TestDB db.I
+}
+
+func (s *DBTestSuite) SetupTest() {
+	s.Assertions = require.New(s.T())
+	s.TestDB = db.ConnectJSONDB("testDB.json")
+	err := s.TestDB.Load()
+	if err != nil {
+		log.Fatalf("Error loading testDB.josn %v\n", err)
+	}
+}
+
+func (s *DBTestSuite) TearDownTest() {
+	err := os.Remove("testDB.json")
+	if err != nil {
+		log.Fatalf("Cannot remove test testDB.json %v\n", err)
+	}
+}
+
+func (s *DBTestSuite) TestQueryAll_EmptyDB() {
+	tasks, found := s.TestDB.QueryAll()
+	require.Equal(s.T(), []common.Task(nil), tasks, "QueryAll shoud return an empty slice of Tasks")
```

Comment:
Why u use the global `require` functions when the receiver inherits the `Assertations`?

`s.Equal([]common.Task(nil), tasks, "QueryAll shoud return an empty slice of Tasks")`